pkgs/servers/nosql/neo4j/default.nix

```diff
@@ -4,21 +4,22 @@ with stdenv.lib;
 
 stdenv.mkDerivation rec {
   name = "neo4j-${version}";
-  version = "3.4.10";
+  version = "3.5.2";
 
   src = fetchurl {
     url = "https://neo4j.com/artifact.php?name=neo4j-community-${version}-unix.tar.gz";
-    sha256 = "0wxcwsnnwk08w3zaz67aa93ysrl61lsy41xynq1sy6z31a7gx9jr";
+    sha256 = "0i36vgs6b24bdhckgkhw23g59x1f2zg6h07c73jv55sdmxmcdpn1";
   };
 
   buildInputs = [ makeWrapper jre8 which gawk ];
 
+
   installPhase = ''
     mkdir -p "$out/share/neo4j"
     cp -R * "$out/share/neo4j"
 
     mkdir -p "$out/bin"
-    for NEO4J_SCRIPT in neo4j neo4j-admin neo4j-import neo4j-shell cypher-shell
```

Comment:
Guessing this was removed upstream?

Reply:
yup, this was [dropped in 3.4.11](https://neo4j.com/release-notes/neo4j-3-4-11/), and was considered deprecated a while before that.